taskfiles/js.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
```

Comment:
for all other existing tasks, we are organizing them based in subprojects, so we have `rpk`, `rp`, `dashboard`, etc. so should we rename this `coproc` or something alike?

Reply:
good idea